lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -783,6 +784,8 @@ TypeSystemSwiftTypeRef::GetSwiftified(swift::Demangle::Demangler &dem,
   if (ident.empty())
     return node;
 
+  if (ident.equals("NSURL") || ident.equals("URL"))
+    llvm::errs();
```

Comment:
not sure if you meant to commit this (mentioning in case this causes CI failures)